ViewEnvironment.podspec

```diff
@@ -0,0 +1,22 @@
+require_relative('version')
+
+Pod::Spec.new do |s|
```

Comment:
Any tests to move too?

Reply:
I didn't see any but I thought about adding a simple `.empty` initialization test.